help/troubleshooting/database/changes-in-the-database-are-not-reflected-on-the-storefront.md

```diff
@@ -61,10 +61,18 @@ To set up the missing MySQL table triggers, you need to re-set the indexer mode:
 
 Use the following command to perform this operation.
 
+>[!WARNING]
+>
+>Before switching indexer modes, we recommend putting your website to [maintenance](../../installation/tutorials/maintenance-mode.md) mode and [disable cron jobs](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure/app/properties/crons-property.html#disable-cron-jobs). This ensures you do not suffer database locks.
```

Comment:
```suggestion
>Before switching indexer modes, we recommend putting your website in [maintenance](https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/setup/application-modes.html#maintenance-mode) mode and [disable cron jobs](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure/app/properties/crons-property.html#disable-cron-jobs) to avoid database locks.
```

Reply:
I suggested a link to the page where we describe application modes for on-prem, but I was tempted to link to the cloud guide that describes the deployment phase. The deployment phase for cloud projects automatically puts a site in maintenance mode according to the docs: https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/develop/deploy/process.html#deploy-phase.

Reply:
Makes sense not to link with ACC only, since the indexer mode change operation should be hosting type agnostic.